gettingStartedWithAstraBlock/README.md

```diff
@@ -12,9 +12,27 @@ Let's get started!
 None
 
 ## 1 Introduction to Astra Block
-With Astra Block, you can stream real-time blockchain data directly to your Astra database. Simply enable the feature within Astra, create a blockchain database and start receiving data.
+With Astra Block, you can stream real-time blockchain data directly to your Astra database. Simply request access to the feature within Astra (we'll enable it for your account within 24 hours) and you can instantly create a blockchain database that auto updates as new blocks are mined.
+
+We currently support Ethereum Mainnet and testnets, with Polygon and Bitcoin coming soon. Interested in support for other chains? [Let us know](mailto:blockchain@datastax.com).
+
+We currently support the following data models:
```

Comment:
We support more tables than these listed.

Block numbers by date
Event signatures
Assets (historical balance snapshots of tokens)